### PR TITLE
SAAS-1065 set default sorting

### PR DESCRIPTION
**Pull request by MaksimMalakhovSimbirsoft**
- added default sorting on columns ticket list table;
Снимок экрана от 2022-08-16 11-16-06

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -9,15 +9,15 @@
...
     "isolatedModules": true,
-    "jsx": "react",
+    "jsx": "react-jsx",
     "lib": [
       "DOM",
       "DOM.Iterable",
       "ES2015"
     ],
-    "module": "commonjs",
+    "module": "esnext",
```
Comment (fabio-silva) [inline tsconfig.json:18]: I believe these changes are unintentional

File: src/components/Table/Table.tsx [hunk 2]
```diff
@@ -35,6 +35,16 @@ export const Table: FC<TableProps> = ({
 }) => {
   const style = useStyles(getStyles);
   const manualPagination = !!(totalPages && totalPages >= 0);
+  const sortees = React.useMemo(
+    () => [
+      {
+        id: columns[0].accessor,
```
Comment (fabio-silva) [inline src/components/Table/Table.tsx:41]: let's just add a safety check `columns[0]?.accessor`

File: src/components/Table/Table.tsx [hunk 3]
```diff
@@ -123,12 +143,15 @@ export const Table: FC<TableProps> = ({
...
+                              onSetActiveIndexSort(index);
+                              setActiveSort(!activeSort);
+                            }}
                           >
                             {column.render('Header')}
                             <span>
-                              {column.isSorted
-                                ? sortedDescCol
-                                : ''}
+                              {activeSort ? ascChevron : descChevron}
```
Comment (fabio-silva) [inline src/components/Table/Table.tsx:154]:
```suggestion
<i className={`fa fa-chevron-${activeSort ? 'up' : 'down'}${index === indexSort ? style.chevronActiveSort : style.chevronSort}`} aria-hidden="true" />
```

**Review by fabio-silva — CHANGES_REQUESTED** [review]

**Review by fabio-silva — APPROVED** [review]

**Review by fabio-silva — APPROVED** [review]

File: tsconfig.json (hunk 1 above)
Comment (MaksimMalakhovSimbirsoft) [reply tsconfig.json]: I roll back changes in ts config

File: src/components/Table/Table.tsx (hunk 3 above)
Comment (MaksimMalakhovSimbirsoft) [reply src/components/Table/Table.tsx]: Looks good. Done

**Review by TomasBlazek3PG — APPROVED** [review]

**Pull request merged**